src/models/unit_tests/rule_test.rs

```diff
@@ -104,7 +104,9 @@ fn test_get_edit_positive_recursive() {
 
   let mut rule_store = RuleStore::default();
 
-  let args = PiranhaArgumentsBuilder::default().build();
+  let args = PiranhaArgumentsBuilder::default()
+    .path_to_codebase("/some/test/path".to_string())
```

Comment:
Worth extracting this into a constant? 

Also, if the point is it won't ever be checked/used...

```
const UNUSED_CODE_PATH="/dev/null"
```

?